.github/workflows/pr-labeler-config-validator.yml

```
@@ -1,19 +1,22 @@
 name: Validate PR Labeler Configuration
 on:
-  push: {}
+  push:
   pull_request:
```

I think it's probably cleaner to just use GitHub Actions built-in rules for this, see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-affects-specific-files: 

```suggestion
  push:
    paths:
      - .github/workflows/pr-labeler-config-validator.yml
      - .github/workflows/pr-labeler.yml
      - .github/pr-labeler.yml
  pull_request:
    paths:
      - .github/workflows/pr-labeler-config-validator.yml
      - .github/workflows/pr-labeler.yml
      - .github/pr-labeler.yml
```

---

🤦‍♂️  I had a vague idea something like this existed as I had used it somewhere long ago. But few stackoverflow results gave the other action. Will change.

---

> But few stackoverflow results gave the other action.

I think it's because the GitHub Actions built-in filters operate on the **entire** `.yml` file, which is fine in our case, since our action does only one thing: validates the PR labeler config.

The `dorny/paths-filter` action lets you use it only a specific step, which is great if you have a much bigger `.yml` file that does a lot of things